.travis.yml

```diff
@@ -17,13 +17,14 @@
 
 language: python
 python:
+  - 3.7
```

Comment:
We should tests 3.8.

Reply:
will remove travis in favor of gha, testing 3.6-3.8 there